doc/transcodingoptions.md

```diff
@@ -0,0 +1,157 @@
+# Configuring Transcoding Options
+
+The Livepeer node offers several methods to configure the transcoding output.
+
+* Webhook authentication. See the [webhook documentation](rtmpwebhookauth.md) for details.
+
+* `-transcodingOptions` CLI flag with a list of presets
+
+* `-transcodingOptions` CLI flag with a JSON configuration file
+
+* `/setBroadcastConfig` endpoint for the CLI API
+
+* `livepeer_cli` tool
+
+### Codecs
+
+Transcoding to and from H.264 is supported.
+
+### Aspect Ratio
+
+The Livepeer transcoder maintains the aspect ratio of the source video in order
+to maintain output video quality. This may sometimes result in the transcoded resolution being somewhat different from the original specification.
+
+ If the aspect ratio of the source video does not match the target aspect ratio, the output is fit to the larger dimension, and the smaller dimension is rescaled proportionately to maintain the source aspect ratio.
+
+If a different behavior is needed, please [let us know](https://github.com/livepeer/go-livepeer/issues/new?template=feature_request.md) by filing a feature request.
+
+### Webhook Authentication
+
+See the [webhook documentation](rtmpwebhookauth.md) for details. To configure the transcoding output, either the `profiles` or `presets` fields in the webhook response can be set, or both.
+
+.se  The `profiles` field in the webhook follows the JSON options that are described in this document, The valid `preset` names follows as described.
```

Comment:
Extraneous ".se"?

Reply:
Yeah that whole line is nonsense. Fixed in bf686d88df7a7bc34da1baca46351a730d83fa37